packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -408,7 +408,7 @@ export class SelectionZone extends React.Component<ISelectionZoneProps, ISelecti
 
         this._onInvokeMouseDown(ev, index);
 
-        const skipPreventDefault = onItemContextMenu(selection.getItems()[index], index, ev.nativeEvent);
+        const skipPreventDefault = onItemContextMenu(selection.getItems()[index], index, (ev as unknown) as Event);
```

Comment:
this would need a big comment explaining whats up here.

Reply:
This really should have been the React event in the first place. Not sure why the native even was passed.